tests/test-query.ipynb

```diff
@@ -38,50 +38,35 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "maap = MAAP(maap_host=populate_maap_api_host(stage))\n"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 4,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "response = maap.executeQuery(\n",
-    "     # dictionary-like object specifying the dataset to query\n",
-    "    src={\n",
+    "if stage != \"dit\": # SKIP until dit is fixed\n",
```

Comment:
This doesn't appear to be in a comment block? Unless I have misunderstood the readme

Reply:
The intention here is that when the deployed services for a stage are broken wrt a test, that we skip running those tests against that stage. Here, Query services are broken for dit, so these tests are ONLY run in production (and presumably staging when we actually have that deployed as a stage)

Reply:
Doy! Apologies. More caffeine required before I comment next time.